magda-web-client/public/index.html

```diff
@@ -1,20 +1,43 @@
 <!doctype html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <link href="https://fonts.googleapis.com/css?family=Rubik:300,400,500,900" rel="stylesheet">
-    <link href="//cdn.muicss.com/mui-0.9.33/css/mui.min.css" rel="stylesheet" type="text/css" media="screen" />
-    <link rel="sitemap" type="application/xml" title="Sitemap" href="/sitemap" />
+    <link href="//cdn.muicss.com/mui-0.9.33/css/mui.min.css" rel="stylesheet" type="text/css" media="screen"/>
+    <link rel="sitemap" type="application/xml" title="Sitemap" href="/sitemap"/>
+    <script>
+        var _rollbarConfig = {
```

Comment:
Minor, but is it possible to put this in `third-party.js` too? I'm hoping one day we'll either migrate away from VWO or it'll stop inserting inline javascript so I can inline JS in the CSP.

Reply:
The Create React App magic variable replacement only happens in HTML files https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#referencing-environment-variables-in-the-html

I tried using process.env.* but third-party.js isn't post-processed because it's an asset not in the /src folder